index.html

```diff
@@ -12,6 +12,22 @@
                 <img class="logo" src="logo.png" />
                 <span class="text">Prechu</span>
             </div>
+            <div class="navigation">
+                <ul class="navbar-nav">
+                    <li class="navbar-nav-item">
+                        <span class="navbar-nav-text">Home</span>
```

Comment:
I think, each item we should have a link, let user be able to click on that menu item